src/main/java/seedu/duke/TransactionManager.java

```diff
@@ -51,4 +51,11 @@ public void populateTransactions (ArrayList<Transaction> transactions) {
         this.transactions = transactions;
     }
 
+    public ArrayList<Transaction> getTransactionList () {
+        return transactions;
```

Comment:
We should not be returning the internal list! It returns by reference so there could be unintended mutations or side effects.